org.mwc.debrief.multipath2/src/org/mwc/debrief/multipath2/MultiPathView.java

```diff
@@ -91,7 +87,9 @@ public MultiPathView()
 	 */
 	public void createPartControl(final Composite parent)
 	{
-		_ui = new MultiPathUI(parent, SWT.EMBEDDED);
+	  parent.setLayout(new FillLayout());
+		_ui = new MultiPathUI(parent, SWT.EMBEDDED);  
+		// SWT.NONE allows some controls to appear - but not graph

```

Comment:
I think that we should be putting the Chart control into a ChartComposite rather than a plain Composite.  I'm playing with that now.